packages/blade/src/components/Input/SelectInput/SelectInput.tsx

```diff
@@ -0,0 +1,79 @@
+import { BaseInput } from '../BaseInput';
+import type { BaseInputProps } from '../BaseInput';
+import type { IconComponent } from '~components/Icons';
+import { ChevronDownIcon, ChevronUpIcon } from '~components/Icons';
+import Box from '~components/Box';
+import { VisuallyHidden } from '~components/VisuallyHidden';
+import { getPlatformType } from '~utils';
+type SelectInputProps = Pick<
+  BaseInputProps,
+  | 'label'
+  | 'labelPosition'
+  | 'necessityIndicator'
+  | 'validationState'
+  | 'helpText'
+  | 'errorText'
+  | 'successText'
+  | 'name'
+  | 'isDisabled'
+  | 'isRequired'
+  | 'prefix'
+  | 'suffix'
+  | 'autoFocus'
+> & {
+  icon?: IconComponent;
+};
+
+const SelectInput = (props: SelectInputProps): JSX.Element => {
+  /**
+   * @TODO handle expanded state with Dropdown
+   */
+  const isPopupExpanded = false;
+  const value = '';
+
+  const { icon, ...baseInputProps } = props;
+
+  const platform = getPlatformType();
+
+  return (
+    <Box position="relative">
+      {/* @TODO Use this for form submissions */}
+      {platform !== 'react-native' ? (
+        <VisuallyHidden>
+          <input
+            tabIndex={-1}
+            required={props.isRequired}
+            value={value}
+            // Accessibility is covered in the select input itself so we hide this field from a11y tree
+            aria-hidden={true}
+          />
+        </VisuallyHidden>
+      ) : null}
+      <BaseInput
+        {...baseInputProps}
+        as="button"
+        textAlign="left"
```

Comment:
do we always default this to left? because on `TextInput` we let consumers decide the alignment so i think we shall follow the same behavior? otherwise in a form where all the input fields have content aligned other than left then this will look weird.

Reply:
I don't see us using `textAlign` on any input other than `TextInput`. What was the reason to support it in TextInput? wouldn't center aligned select look a bit strange?